pyproject.toml

```diff
@@ -89,6 +89,4 @@ user = [
 where = ["./"]
 
 [tool.setuptools.package-data]
-pycqed = [
-    "utilities/WARNING.png",
-]
+pycqed = ["utilities/WARNING.png", "**/*.json"]
```

Comment:
This is not needed when installing in editable mode, which we usually do at qudev, but it is useful to change it to also allow installation in non-editable mode.